tests/conftest.py

```diff
@@ -118,18 +117,6 @@ def gprofiler_docker_image(docker_client: DockerClient) -> Iterable[Image]:
     yield docker_client.images.get("gprofiler")
 
 
-@fixture(scope="session")
-def gprofiler_docker_image_resources(gprofiler_docker_image: Image):
```

Comment:
hmm, If you removed this fixture, and now you rely that resources will be available on disk, execute the `copy_resource_from_image.sh` in the `test.sh` script

Reply:
I don't want it to run implicitly because the user might need to supply the image name. It's now explained in README for local installation, so I think it's alright

Reply:
I see, let's do something else, let's check in the `test.sh` script that the resources are available prior to running the pytest test suite and if not prints a message and exits

Reply:
kk